wagtailio/project_styleguide/templates/patterns/components/buttons/button.html

```diff
@@ -0,0 +1,6 @@
+<a href="{{ url }}" class="button {% if classes %}{{ classes }}{% endif %}">
+    <p class="button__text">{{ title }}</p>
+    {% if arrow %}
+        {% include "patterns/atoms/svgs/arrow.html" with classes="arrow" %}
```

Comment:
wasn't the idea of `{% include "includes/svg_sprites.html" %}` to include all svg icons, so you can just reference them here?

Reply:
It was, I was thinking of taking a mixed approach but realised realistically we're doubling effort. I've added the SVG to the sprites file instead.